bigquery/tables.js

```diff
@@ -88,29 +204,73 @@ var program = module.exports = {
 };
 
 cli
-  .command('export <bucket> <file> <dataset> <table>', 'Export a table from BigQuery to Google Cloud Storage.', {
+  .demand(1)
+  .command('create <dataset> <table>', 'Create a new table the specified dataset.', {}, function (options) {
+    program.createTable(options, makeHandler());
+  })
+  .command('list <dataset>', 'List tables in the specified dataset.', {}, function (options) {
+    program.listTables(options, makeHandler(true, 'id'));
+  })
+  .command('delete <dataset> <table>', 'Delete a table in the specified dataset.', {}, function (options) {
+    program.deleteTable(options, makeHandler());
+  })
+  .command('import <dataset> <table> <file>', 'Import data from a local file or a Google Cloud Storage file into BigQuery.', {
+    bucket: {
+      alias: 'b',
+      requiresArg: true,
+      description: 'Specify Cloud Storage bucket.',
+      type: 'string'
+    },
+    format: {
+      alias: 'f',
+      requiresArg: true,
+      type: 'string',
+      choices: ['JSON', 'CSV', 'AVRO']
+    }
+  }, function (options) {
+    program.importFile(options, makeHandler());
+  })
+  .command('export <dataset> <table> <bucket> <file>', 'Export a table from BigQuery to Google Cloud Storage.', {
     format: {
       alias: 'f',
-      global: true,
       requiresArg: true,
       type: 'string',
       choices: ['JSON', 'CSV', 'AVRO']
     },
     gzip: {
-      global: true,
       type: 'boolean',
       description: 'Whether to compress the exported table using gzip. Defaults to false.'
     }
   }, function (options) {
-    program.exportTableToGCS(options, console.log);
+    program.exportTableToGCS(options, makeHandler());
   })
   .example(
-    'node $0 export my-bucket my-file my-dataset my-table',
-    'Export my-dataset:my-table to gcs://my-bucket/my-file as raw JSON'
+    'node $0 create my_dataset my_table',
+    'Create table "my_table" in "my_dataset".'
+  )
+  .example(
+    'node $0 list my_dataset',
+    'List tables in "my_dataset".'
+  )
+  .example(
+    'node $0 delete my_dataset my_table',
+    'Delete "my_table" from "my_dataset".'
+  )
+  .example(
+    'node $0 import my_dataset my_table ./data.csv',
+    'Import a local file into a table.'
+  )
+  .example(
+    'node $0 import my_dataset my_table data.csv --bucket my-bucket',
+    'Import a GCS file into a table.'
+  )
+  .example(
+    'node $0 export my_dataset my_table my-bucket my-file',
```

Comment:
I don't care. Only constraint is that dataset and table IDs _can't_ have hyphens.